releaseNotes/AzureAppConfigurationProviderPython.md

```diff
@@ -0,0 +1,20 @@
+# azure-appconfiguration-provider
+
+[Source code][source_code] | [Package (Pypi)][package] | [Samples][samples]
+
+## 1.0.0b1 - October 13, 2022
+
+Initial Beta Release of the Azure App Configuration Provider for Python
+
+### New Features
+
+* Connecting to an App Configuration Store using a connection string or Azure Active Directory.
+* Selecting multiple sets of configurations using SettingSelector.
+* Trim prefixes off key names.
+* Resolving Key Vault References, requires AAD.
+* Secret Resolver, resolve Key Vault References locally without connecting to Key Vault.
+* Json Content Type
+
+[package]: https://pypi.org/project/azure-appconfiguration-provider/
+[samples]: https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/appconfiguration/azure-appconfiguration-provider/samples
+[source_code]: https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/appconfiguration/azure-appconfiguration-providerspring-cloud-azure-appconfiguration-config
```

Comment:
This link doesn't work for me.

BTW, please add an empty line at the end of the file.